cocos/renderer/pipeline/deferred/DeferredPipeline.cpp

```diff
@@ -128,7 +128,6 @@ bool DeferredPipeline::activate() {
 }
 
 void DeferredPipeline::render(const vector<uint> &cameras) {
-    if (cameras.empty()) return;
```

Comment:
Why remove it?

Reply:
> Why remove it?

ZeQiang made a judgment at the JS layer, it is not needed here. And not calling flushCommands may cause multi-thread blocking.